nodejs/README.md

```diff
@@ -0,0 +1,208 @@
+# Node.js
+
+> [Node.js](https://nodejs.org/) is a JavaScript runtime built on Chrome's V8
+> JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that
+> makes it lightweight and efficient. Node.js' package ecosystem, npm, is the
+> largest ecosystem of open source libraries in the world.
+
+This is a living guide that describes some best practices around using Node.js
+at 18F. For areas we don't have a specific recommendation, we try to document
+what folks are currently using to help inform your decision.
+
+This guide will also help facilitate discussions around these technologies so we
+can identify where there is consensus and where there is disagreement.
+
+For each area, we'll include Recommendations, Suggestions, or Considerations.
+Recommendations are practices where there is a strong consensus on the team.
+Suggestions are practices where there is some agreement but many reasons why you
+might choose something different. Considerations are tips to help you decide for
+yourself what is right for your project.
+
+If you have any questions, hop into the
+[#javascript](https://gsa-tts.slack.com/messages/javascript/) Slack channel.
+
+
+## Twelve-factor web applications
+
+[12factor.net](https://12factor.net/) describes a twelve-factor methodology for
+building scalable web applications. By following the methodology, your
+application implements a generic contract with the platform. This contract
+allows it to be scaled and managed by the platform without intimate knowledge of
+how the application works.
+
+[cloud.gov](https://cloud.gov/) is the preferred platform at 18F. Building your
+Node.js application with the Twelve-factor methodology in mind will ensure your
+application is designed for cloud.gov.
+
+
+### Best practices
+
+#### Recommend
+
+- Build your application on top of cloud.gov.
+- Read the [Twelve-factor methodology](https://12factor.net/) and keep it in
+  mind as you build your application.
+
+
+## Choosing a version
+
+### Node.js
+
+Node.js supports certain versions as Long-term Support (LTS) versions. We
+recommend the [current active LTS](https://github.com/nodejs/LTS#lts-schedule1)
+version, currently v6.x.
```

Comment:
Move "currently v6.x" down to the recommendation of this section so it is easier to see.